frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -129,6 +129,7 @@
     "TASKCFG_ALL_CASSANDRA_STREAMING_SOCKET_TIMEOUT_IN_MS": "{{cassandra.streaming_socket_timeout_in_ms}}",
     "TASKCFG_ALL_CASSANDRA_MAX_VALUE_SIZE_IN_MB": "{{cassandra.max_value_size_in_mb}}",
     "TASKCFG_ALL_CASSANDRA_OTC_COALESCING_STRATEGY": "{{cassandra.otc_coalescing_strategy}}",
+    "TASKCFG_ALL_LOCAL_SEEDS": "node-0-server.{{service.name}}.autoip.dcos.thisdcos.directory,node-1-server.{{service.name}}.autoip.dcos.thisdcos.directory",
```

Comment:
This similarly ignores the seed_count setting. I think this is what the logic in Main.java was for.